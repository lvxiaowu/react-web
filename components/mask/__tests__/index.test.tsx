import React from 'react';
import { render, shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Mask from '../index';

describe('Mask', () => {
  it('renders correctly', () => {
    const wrapper = render(<Mask visible />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });

  it('type is transparent', () => {
    const wrapper = render(<Mask visible type="transparent" />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
