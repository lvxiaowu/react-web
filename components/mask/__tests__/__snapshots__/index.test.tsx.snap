// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Mask renders correctly 1`] = `
<div
  class="sd-mask sd-mask--normal"
/>
`;

exports[`Mask type is transparent 1`] = `
<div
  class="sd-mask sd-mask--transparent"
/>
`;
